src/main/webapp/data-visualization.js

```diff
@@ -508,6 +529,142 @@ function toggleDataTable() {
   }
 }
 
+function displayYearlyChart(chartId, data, description, location) {
+  const chart = am4core.create(chartId, am4charts.XYChart);
+  chart.data = data;
+  const dateAxis = chart.xAxes.push(new am4charts.DateAxis());
+  dateAxis.dataFields.category = 'year';
+  dateAxis.title.text = 'Year';
+  dateAxis.baseInterval = {timeUnit: 'year'};
+  const valueAxis = chart.yAxes.push(new am4charts.ValueAxis());
+  valueAxis.title.text = description;
+  const series = chart.series.push(new am4charts.LineSeries());
+  series.dataFields.valueY = 'value';
+  series.dataFields.dateX = 'year';
+  series.name = description;
+  series.tooltip.getFillFromObject = true;
+  series.tooltipText = '{valueY}';
+  series.strokeWidth = 3;
+
+  const color = getColor();
+  series.stroke = am4core.color(color);
+  series.yAxis = valueAxis;
+
+  const bullet = series.bullets.push(new am4charts.CircleBullet());
+  bullet.stroke = am4core.color('#fff');
+  bullet.fill = am4core.color(color);
+  bullet.strokeWidth = 2;
+  bullet.tooltipText = '{valueY}';
+  bullet.circle.radius = 4;
```

Comment:
I think it would be nice if the bullets were a bit bigger. They would stand out more and it would also be easier to trigger the tooltip text (right now I sometimes mouse past them by accident instead of over them).

Reply:
Good point! Changed radius to 8, let me know if it's still too small

Reply:
I think this is good.